client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,77 @@
+@import "base.scss";
+
+button:hover{
```

Comment:
This is good, but it could be a more specific selector so that there's less of a chance its properties affect other elements! Perhaps moving the `cursor: pointer;` into each of the button classes will do.